src/main/kotlin/io/snyk/plugin/services/SnykApiService.kt

```diff
@@ -23,7 +23,10 @@ import javax.net.ssl.X509TrustManager
 class SnykApiService : Disposable {
 
     val sastSettings: CliConfigSettings?
-        get() = snykApiClient?.sastSettings
+        get() {
+            val pluginSettings = service<SnykApplicationSettingsStateService>()
```

Comment:
Any reason to not make that a field / property?

Reply:
I prefer functions, Artsiom is for properties. So it's 50/50, but for consistency reasons (userMe), I leave it as property.